.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,91 @@
+name: Bug Report
+description: Use this form to report a bug with Citizens.
+
+body:
+  - type: 'markdown'
+    attributes:
+      value: |-
+        ## DO NOT IGNORE
+        Please fill out all required fields the way they are described as.
+        **Not following the issue form will get it closed automatically!**
+        
+        ----
+        
+        ### Section 1) Version Infos
+        Include the outputs of `/version` and `/version Citizens` in the respective fields.
+  - type: 'textarea'
+    attributes:
+      label: '`/version` output'
+      description: 'Include the **complete** output of the `/version` command here.'
+      placeholder: 'This server is running ...'
+      render: txt
+    validations:
+      required: true
+  - type: 'textarea'
+    attributes:
+      label: '`/version Citizens` output'
+      description: 'Include the **complete** output of the `/version Citizens` command here.'
+      placeholder: 'Citizens Version ...'
+      render: txt
+    validations:
+      required: true
+  - type: 'markdown'
+    attributes:
+      value: |-
+        ----
+        
+        ### Section 2) Log and Configuration files
+        Provide your server's latest log file and the configuration file (`config.yml`) of Citizens.
+        
+        When your problem is a Performance issue will you need to also include the `saves.yml` of Citizens.
+  - type: 'input'
+    attributes:
+      label: 'Log file'
+      description: 'Upload your server''s latest.log on pastebin or similar and share the URL.'
+      placeholder: 'https://pastebin.com/...'
+    validations:
+      required: true
+  - type: 'input'
+    attributes:
+      label: 'config.yml'
+      description: 'Upload the config.yml of Citizens on pastebin or similar and share the URL.'
+      placeholder: 'https://pastebin.com/...'
+    validations:
+      required: true
+  - type: 'input'
+    attributes:
+      label: 'saves.yml'
+      description: |-
+        **ONLY REQUIRED FOR PERFOMANCE ISSUES!**
+        Upload the saves.yml of Citizens on pastebin or similar and share the URL.'
+      placeholder: 'https://pastebin.com/...'
+  - type: 'markdown'
+    attributes:
+      value: |-
+        ----
+        
+        ### Section 3) Performance Report
+        **This section is only required when having Performance issues!**
+        
+        Include a report from a profiler such as Spark [[Download](https://spark.lucko.me/download)].
```

Comment:
Link should be https://www.spigotmc.org/resources/spark.57242/

Reply:
I used the spark page since it included multiple platforms and not just spigot alone. Also, releases there should be identical to Spigot ones.

Reply:
Citizens only runs on Spigot and not any of the other platforms. It's just added confusion to no benefit to the user that's posting a Citizens issue